src/elfpy/user.py

```diff
@@ -115,13 +164,17 @@ def get_trade(self):
         and care less about how much we have to spend.
         we spend what we have to spend, and get what we get.
         """
+        self.status_update()
         action_list = self.action()  # get the action list from the policy
         for action in action_list:  # edit each action in place
             if action.mint_time is None:
                 action.mint_time = self.market.time
             if action.action_type == "close_short":
                 action.token_in_protocol = self.wallet.token_in_protocol[action.mint_time]
                 action.base_in_protocol = self.wallet.base_in_protocol[action.mint_time]
+            if action.action_type == "close_short":
```

Comment:
why is this conditional listed twice now?

Reply:
Removed the repeated conditional block